### ISSUE TITLE
Losing checkable state when went back

### ISSUE THREAD

**Issue by Tarcisiofl** [albinpoignot/CheckableCardView#1]
Hello Albin,

I have implemented the Checkable CardView.
However, if the user return to the page with it, no item is checked.

How can I keep it?

**Comment by albinpoignot** [comment]
Hi Tarcisiofl,

**TL;DR;** _"Manually" set the status to the one you want when resuming the screen (Activity or Fragment or whatever)._

The "checked" status should not be hold by your UI component. The UI component is responsible only for drawing something on screen.

https://github.com/albinpoignot/CheckableCardView/blob/bf515b5910ec3a73f85b344b9a4dd2a42f2b7616/app/src/main/java/com/example/checkablecardview/MainActivity.java#L16

You can see above how to set the checked status. When coming back on the screen, you should restore the status yourself (somewhere like in the `.onResume(...)` method). 

Hope it helps.
